releases/2021.02.yaml

```diff
@@ -179,3 +179,7 @@ repositories:
     type: git
     url: https://github.com/dic-iit/matio-cpp.git
     version: v0.1.0
+  diagnosticdaemon:
+    type: git
+    url: https://github.com/robotology/diagnostic-daemon.git
+    version: v1.0.0
```

Comment:
Can you add a newline at the end of the file to avoid Git/GitHub warnings? See https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file .